scripts/utils/create_package_template.py

```diff
@@ -37,6 +37,16 @@
     "Utilities",
     "VB",
     "Web Application",
+    "Active Directory Tools",
+    "Command & Control",
+    "Evasion",
+    "Exploitation",
+    "Information Gathering",
+    "Networking Tools",
+    "Password Attacks",
+    "Utilities",
+    "Vulnerability Analysis",
+    "Wordlists"
```

Comment:
Removed redundant "Networking Tools". Also do we need the word "Tools"?

```suggestion
    "Active Directory",
    "Command & Control",
    "Evasion",
    "Exploitation",
    "Information Gathering",
    "Password Attacks",
    "Utilities",
    "Vulnerability Analysis",
    "Wordlists"
```